openwisp_controller/config/migrations/0018_device_unique_together_constraints.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0.9 on 2018-12-19 14:30
+
+import django.core.validators
+from django.db import migrations, models
+import re
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0004_default_groups'),
+        ('config', '0017_template_name_organization_unique_together'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
this `AlterField` and the next one can be removed if we avoid redefining `name` and `mac_address`, which is something I think we should do